Client/Frontend/Strings.swift

```diff
@@ -1624,6 +1624,47 @@ extension String {
                     comment: "Title for the page where the user sign in to their Firefox Sync account.")
             }
         }
+
+        public struct Search {
+            public static let Title = MZLocalizedString(
+                key: "Settings.Search.PageTitle.v121",
+                tableName: "Settings",
+                value: "Search",
+                comment: "Navigation title for search page in the Settings menu.")
+            public static let ShowSearchSuggestions = MZLocalizedString(
+                key: "Settings.Search.ShowSuggestions.v121",
+                tableName: "Settings",
+                value: "Show Search Suggestions",
+                comment: "Label for show search suggestions setting, in the Search Settings page.")
+            public static let DefaultSearchEngineTitle = MZLocalizedString(
+                key: "Settings.Search.DefaultSearchEngine.Title.v121",
+                tableName: "Settings",
+                value: "Default Search Engine",
+                comment: "Title for default search engine settings section in the Search page in the Settings menu.")
```

Comment:
Small suggestion again, it just feels like it reads better that way if you don't know what this is about. I won't make the comments on the other strings, but I would make the same comment 🤷‍♀️
```suggestion
                comment: "Title for the `default search engine` settings section in the Search page in the Settings menu.")
```